hedera-node/hedera-admin-service-impl/src/main/java/com/hedera/node/app/service/admin/impl/handlers/FreezeHandler.java

```diff
@@ -71,24 +83,24 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
         }
 
         final FreezeType freezeType = freezeTxn.freezeType();
+        final var txValidStart = context.body().transactionID().transactionValidStart();
```

Comment:
Did this line give you warnings? I would have expected it to require "...OrThrow" variants of some of these method calls.

Reply:
Yep, I found a warning. This was most recently modified via one of the merges and I don't think I had checked for warnings after the merge.